### PR TITLE
tests: add qemu to our spread.yaml and update README

### PR DESCRIPTION
**Pull request by mvo5**

### CODE REVIEW

File: spread.yaml [hunk 1]
```diff
@@ -14,8 +14,13 @@ backends:
...
             - ubuntu-16.04-64-grub
             - ubuntu-16.04-32-grub
+    qemu:
+        systems:
+            - ubuntu-16.04:
+                username: ubuntu
+                password: ubuntu
 
-path: $[PROJECT_PATH]
+path: /home/gopath/src/github.com/snapcore/snapd
```
Comment (fgimenez) [inline spread.yaml]: Can't we use here `$(PROJECT_PATH)` so that we don't need to repeat it?

**Comment by fgimenez** [comment]
:+1: with small ignorable nitpick, thanks! :)

File: spread.yaml (hunk 1 above)
Comment (mvo5) [inline spread.yaml]: That is what I had initially, but it errors with:
`error: remote project path must be absolute and not /: $(PROJECT_PATH)`

File: spread.yaml [hunk 2]
```diff
@@ -1,21 +1,26 @@
 project: snapd
 
 environment:
-    REUSE_PROJECT: $(echo $REUSE_PROJECT)
+    REUSE_PROJECT: "$(HOST: echo $REUSE_PROJECT)"
     PROJECT_PATH: /home/gopath/src/github.com/snapcore/snapd
     GOPATH: /home/gopath
     PATH: /home/gopath/bin:/snap/bin:$PATH
-    TESTSLIB: $[PROJECT_PATH]/tests/lib
+    TESTSLIB: /home/gopath/src/github.com/snapcore/snapd/tests/lib
```
Comment (mvo5) [inline spread.yaml]: I'm slightly unhappy about this part here, but I guess with `$[]` going away there is no way to avoid it?

Comment (niemeyer) [inline spread.yaml]: This can be $(PROJECT_PATH), I think?

File: spread.yaml (hunk 1 above)
Comment (niemeyer) [inline spread.yaml]: This is a slightly tricky one, but maybe we can do something about later.

**Pull request merged**